File: python3/putRequest.py
import urllib3

http = urllib3.PoolManager()

company = "YOUR_TEAMWORK_SITE_NAME"
key = "YOUR_API_KEY"
task_id = "TASK_ID"
action = "tasks/{0}/complete.json".format(task_id)

url = "https://{0}.teamwork.com/{1}".format(company, action)
headers = urllib3.util.make_headers(basic_auth=key + ":xxx")
request = http.request('PUT', url, headers=headers)

response = request.status
data = request.data

print(data)

File: php/putRequest.php
$company = "YOUR TEAMWORK SITENAME";
$key = "YOUR_API_KEY";
$taskId = 12345;
$action = "todo_items/".$taskId."/complete.json";
$channel = curl_init();
curl_setopt( $channel, CURLOPT_URL, "https://". $company .".teamwork.com/". $action );
curl_setopt( $channel, CURLOPT_RETURNTRANSFER, 1 );
curl_setopt($channel, CURLOPT_CUSTOMREQUEST, "PUT");
curl_setopt( $channel, CURLOPT_HTTPHEADER,
array( "Authorization: BASIC ". base64_encode( $key .":xxx" ))
);
echo curl_exec ( $channel );
curl_close ( $channel );